modules/geo/src/main/java/org/opensearch/geo/search/aggregations/bucket/geogrid/InternalGeoTileGridBucket.java

```diff
@@ -52,8 +53,8 @@ class InternalGeoTileGridBucket extends BaseGeoGridBucket<InternalGeoTileGridBuc
     /**
      * Read from a stream.
      */
-    InternalGeoTileGridBucket(StreamInput in) throws IOException {
-        super(in);
+    InternalGeoTileGridBucket(BaseStreamInput in) throws IOException {
```

Comment:
Same here, keeping `StreamInput` (for now) would be better I think

Reply:
Same as above